backend/sample.js

```diff
@@ -84,11 +102,11 @@ app.post("/:folderName", (req, res) => {
     res.status(404).send("Folder not found.");
   } else {
     if (!result["isPrivate"]) {
-      result = result.notes;
+      result = findNotesByFolder(folderName);
       res.status(201).send(result);
     } else {
       if (passw["password"] === result["password"]) {
-        res.status(201).send(result.notes);
+        res.status(201).send(findNotesByFolder(folderName));
```

Comment:
By checking the password in the "/:folder" endpoint, I think that means we would have to access the folder on the frontend and then submit yet again. Maybe something with app.get("/") would make more sense

Reply:
Make sense. I'll improve this part.